### ISSUE TITLE
Possible typo?

### ISSUE THREAD

**Issue by gaceladri** [NVIDIA-Merlin/Merlin#67]
https://github.com/NVIDIA-Merlin/Merlin/blob/5e870d7078232bdd895c349cef4f118d1f4ad73d/docker/dockerfile.torch#L45

Hi, is it possible that here we have a duplicated pip install?

Thanks!

**Comment by gaceladri** [comment]
No, if it makes sense for you, it is right. I though **that pip install pip install repeated two times** was an error. Closing if it makes sense to you. :+1:

**Comment by gaceladri** [comment]
I think that you dont noticed that your repeating pip install two times... This is what pip install two times does, it installs "install" and not scikit-learn
```
ad@ad-mol:~$ pip install pip install scikit-learn
Requirement already satisfied: pip in ./anaconda3/envs/nvt/lib/python3.8/site-packages (21.3.1)
Collecting install
  Using cached install-1.3.4-py3-none-any.whl (3.1 kB)
Requirement already satisfied: scikit-learn in ./anaconda3/envs/nvt/lib/python3.8/site-packages (1.0.1)
Requirement already satisfied: numpy>=1.14.6 in ./anaconda3/envs/nvt/lib/python3.8/site-packages (from scikit-learn) (1.20.3)
Requirement already satisfied: scipy>=1.1.0 in ./anaconda3/envs/nvt/lib/python3.8/site-packages (from scikit-learn) (1.7.2)
Requirement already satisfied: threadpoolctl>=2.0.0 in ./anaconda3/envs/nvt/lib/python3.8/site-packages (from scikit-learn) (3.0.0)
Requirement already satisfied: joblib>=0.11 in ./anaconda3/envs/nvt/lib/python3.8/site-packages (from scikit-learn) (1.1.0)
Installing collected packages: install
Successfully installed install-1.3.4
```

**Comment by benfred** [comment]
Oh you're right! Sorry I misread your initial comment

**Comment by benfred** [comment]
Thanks for pointing this out! I have a fix in #68 - (I was thinking you were saying we 'pip install' gevent twice, rather than we had pip install twice in the that line 🤦 )

**Comment by gaceladri** [comment]
A small typo in the ocean :) keep doing the great work :) :+1:
